specs/pvs/corla.pvs

```diff
@@ -92,35 +139,9 @@
 %%% \item[github] the URL for the GitHub project containing an artifact
 %%% \end{description}
 %%%
-%%% \section{System Overview}
-%%%
-%%% The RLA Tool is being developed by Free \& Fair for the Colorado
-%%% Department of State (CDOS henceforth). The RLA Tool facilitates
-%%% running a risk limiting audit across several jurisdictions. In the
-%%% case of Colorado, it facilitates running risk limiting audits
-%%% across all counties in the state simultaneously.
-%%%
-%%% The Colorado election law pertaining to election audits is
-%%% \href{http://law.justia.com/codes/colorado/2016/title-1/general-primary-recall-and-congressional-vacancy-elections/article-7/part-5/section-1-7-515/}
-%%% {Section 1-7-515, C.R.S.}  and the rules pertaining to such are
-%%% found in the "Revised Draft of Proposed Rules" whose latest
-%%% version is labeled CCR 1505-1 and dated July 6, 2017. Section 25,
-%%% Post-Election Audit, of that document is the salient portion of
-%%% that document. We will call this document RDPR-6-Jul-2017 for the
-%%% purpose of traceability in this specification.
-%%%
-%%% This domain model is based upon that law, its rules, our contract,
-%%% our bibliography of publications about risk-limiting audits, etc.
-
-
-% @todo kiniry Add a discussion of our glossary and its relationship
-% to this specification.
-
-% @todo kiniry Define C.R.S. and explain the other sources used to
-% derive all concepts in this specification.
-
 % @todo Explain how refinement works and how the reader can understand
 % exactly what is built vs. what is just designed.
+% @review kiniry what's the point of including these annotations? I don't see them used in the document @sfsinger
```

Comment:
Which annotations do you mean? The @todo above, or the Literate PVS notes below? Both seem used and useful to me.